cattle/plugins/docker/compute.py

```diff
@@ -92,25 +92,86 @@ def _add_instances(self, ping, pong):
         if not utils.ping_include_instances(ping):
             return
 
+        utils.ping_add_resources(pong, {
+            'type': 'hostUuid',
+            'uuid': DockerConfig.docker_uuid()
+        })
+
         containers = []
-        for c in docker_client().containers():
-            names = c.get('Names')
-            if names is None:
-                continue
-
-            for name in names:
-                if name.startswith('/'):
-                    name = name[1:]
-                    containers.append({
-                        'type': 'instance',
-                        'uuid': name,
-                        'state': 'running',
-                        'dockerId': c.get('Id')
-                    })
+        all, stopped = self._get_all_containers_by_state()
```

Comment:
should we rename "all" var? As the current name implies that it container all the containers, but it excludes all the containers belonging to "stopped" var